automation/run-tests.sh

```diff
@@ -234,8 +234,16 @@ function remove_tempdir {
    rm -rf "$NMSTATE_TEMPDIR"
 }
 
+function vdsm_tests {
+  trap remove_tempdir EXIT
```

Comment:
Please, use 4 spaces for each indentation. Thanks!

Reply:
Done, thanks